src/test/app/NFToken_test.cpp

```diff
@@ -5079,13 +5112,15 @@ class NFToken_test : public beast::unit_test::suite
         using namespace test::jtx;
         FeatureBitset const all{supported_amendments()};
         FeatureBitset const fixNFTDir{fixNFTokenDirV1};
+        FeatureBitset const fixNFTBroker{fixUnburnableNFToken};
```

Comment:
let's get rid of this - it will cause additional test runs for no real benefit, since we are using the same feature flag

Reply:
Gone